tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/LabelGetMethod.scala

```diff
@@ -71,7 +71,7 @@ class LabelMethodModule extends AbstractModule {
   @Named("labelChangeDefaultLimit")
   @Singleton
   def provideLimitDefault(propertiesProvider: PropertiesProvider): Limit =
-    Limit.of(Option(propertiesProvider.getConfiguration("jmap"))
+    Limit.of(Try(propertiesProvider.getConfiguration("jmap")).toOption
       .map(configuration => configuration.getInt("labelChangeDefaultLimit", DEFAULT_LIMIT_LABEL_CHANGES))
```

Comment:
Q: Do we really need to make this configurable?

For email/mailbox change limit we just hard code
```
        bind(Limit.class).annotatedWith(Names.named(CassandraEmailChangeRepository.LIMIT_NAME)).toInstance(Limit.of(256));
        bind(Limit.class).annotatedWith(Names.named(CassandraMailboxChangeRepository.LIMIT_NAME)).toInstance(Limit.of(256));
````

Reply:
If it should not be configurable, it will be better if we use a static final variable. 
Too verbose when creating an instance bean for Limit and set static is `Limit.of(256)`

Reply:
We decided back then to have a a code defined limit and not configurable, not worth the trouble for now.

I agree though that the all system should use the same limit then and would be ok for a refactoring to have a constant for it.